packages/backend-function/src/factory.ts

```diff
@@ -115,6 +115,13 @@ export type FunctionProps = {
    */
   memoryMB?: number;
 
+  /**
+   * The size of the function's /tmp directory in MiB.
+   * Must be a whole number.
+   * Default is 512MiB.
```

Comment:
```suggestion
   * @default 512
```

Reply:
@josefaidt 
Thank you for your review and suggestion.
I have modified it as you suggested.